src/qt/coincontroldialog.h

```diff
@@ -63,7 +63,7 @@ class CoinControlDialog : public QDialog
 
     QMenu *contextMenu;
     QTreeWidgetItem *contextMenuItem;
-    QAction *copyTransactionHashAction;
+    QAction *copyTransactionOutpointAction;
```

Comment:
The Clang format checker suggested the following change:

```suggestion
    QAction* copyTransactionOutpointAction;
```

Reply:
nit, also use `m_` prefix and snake case.